koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -605,7 +605,7 @@ def _get_labels_override(self, opts):
         labels = {}
         release = opts.get('release', None)
         if release:
-            labels['release'] = release
+            labels['Release'] = release
```

Comment:
No, I think the rule until we switch to the new label set completely is:
* prefer the new set when reading labels
* use the old set when writing labels